src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, region_silo_model, sane_repr
+
+from .data_condition_group import DataConditionGroup
+
+
+@region_silo_model
+class DataCondition(DefaultFieldsModel):
+    """
+    A data condition is a way to specify a logic condition, if the condition is met, the condition_result is returned.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+    __repr__ = sane_repr("type", "condition")
+
+    condition = models.CharField(max_length=200)
+    threshold = models.FloatField(blank=True, null=True)
+    comparison = models.CharField(max_length=200, blank=True, null=True)
+
```

Comment:
Wondering if it's worthwhile to have comments here explaining what the values look like.